Samples/Xamarin/GetStartedXamarinAndroid/GetStartedXamarinAndroid/MyFirebaseMessagingService.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
```

Comment:
Same as above, these `using` statements should be sorted and those not used should be removed.